File: CS50/phonebook/phonebook.py
from cs50 import get_string
name = get_string("What is your name?")

# This is a dictionry, aka: dict
people = {
    "Carter": "+1-617-495-1000",
    "David": "+1-949-468-2750"
}

name = get_string("Name: ")
if name in people:
    print(f"Number: {people[name]})

File: CS50/phonebook/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    FILE* file = fopen("phonebook.csv", "a"); //a stands for append mode, asterisk can move to the other side
    if (file == NULL)
    {
        printf("Unable to open the file\n");
        return 1;
    }

    string name = get_string("Name: ");
    string number = get_string("Phone number: ");

    fprintf(file, "%s, %s\n", name, number);
    fclose(file); // always close your file.

}